data/src/main/java/com/puzzlebench/cmk/data/service/response/ComicsResponse.kt

```diff
@@ -0,0 +1,6 @@
+package com.puzzlebench.cmk.data.service.response
+
+class ComicsResponse (
```

Comment:
please do not initialize variables in NULL in kotlin, only do that when is necessary

Reply:
okey, all fields null were changed with default values